src/vs/workbench/contrib/terminal/browser/links/terminalLinkManager.ts

```diff
@@ -163,10 +167,11 @@ export class TerminalLinkManager extends DisposableStore {
 		return { wordLinks, webLinks, fileLinks };
 	}
 
+	// TODO: Convert to use ITerminalSimpleLink
 	async getLinksForType(y: number, type: 'word' | 'web' | 'file'): Promise<ILink[] | undefined> {
 		switch (type) {
 			case 'word':
-				return (await new Promise<ILink[] | undefined>(r => this._standardLinkProviders.get(TerminalWordLinkProvider.id)?.provideLinks(y, r)));
+				return (await new Promise<ILink[] | undefined>(r => this._standardLinkProviders.get(TerminalWordLinkDetector.id)?.provideLinks(y, r)));
 			case 'web':
 				return (await new Promise<ILink[] | undefined>(r => this._standardLinkProviders.get(TerminalProtocolLinkProvider.id)?.provideLinks(y, r)));
```

Comment:
I think you're doing just word only in this PR, otherwise, should replace protocol here?

Reply:
I'm doing all, I missed this part. Soon I will delete all the old providers and fix up their tests at which point this will result in a compile error